GUI/DeusEx/Classes/PersonaScreenBingo.uc

```diff
@@ -98,6 +99,24 @@ function bool ResetBingoBoard()
     return true;
 }
 
+function ShowBingoGoalHelp( Window bingoTile )
+{
+    local BingoTile bt;
+    local BingoHintMsgBox msgbox;
+
+    bt=BingoTile(bingoTile);
+
+    if(bt==None){
+        //Don't know how we got here, but might as well check
+        return;
+    }
+    msgbox = BingoHintMsgBox(root.PushWindow(class'BingoHintMsgBox',False));
+    msgbox.SetTitle(bt.GetText());
+    msgbox.SetMessageText(bt.GetHelpText());
```

Comment:
what are you thinking for these help texts? gonna put them in the defaultproperties in DXREvents? or gonna make a function with a big switch/case?